apidaze/sipusers/__init__.py

```diff
@@ -0,0 +1,250 @@
+from apidaze.http import Http, HttpMethodEnum
+
+
+class Sipusers(object):
+    """
+        Initializes the Sipusers class
+
+        Parameters
+        ----------
+        http: Http
+            Apidaze Http object
+
+        Returns
+        -------
+        object
+            The Apidaze Sipusers object
+    """
+    def __init__(self, http: Http):
+        self.http = http
+        self.endpoint = '/sipusers'
+
+    def list(self):
+        """
+            Shows a list of existing SIP users
+
+            Returns
+            -------
+            dict
+                JSON response
+        """
+        response = self.http.request(
+            method=HttpMethodEnum.GET,
+            endpoint=self.endpoint,
+            payload={}
+            )
+
+        result = {
+            'body': response.json(),
+            'status_code': response.status_code
+        }
+
+        return result
+
+    def create(self, username: str, name: str,
+               email_address: str,
+               internal_caller_id_number: str,
+               external_caller_id_number: str):
+        """
+            Create a SIP User for devices to register
+            to Apidaze Freeswitches
```

Comment:
Do you think it'd be better to use generic names for some parts like `FreeSWITCH`?

```suggestion
            to Apidaze SIP servers
```

Reply:
Copy & paste from documentation :)